src/pudl/metadata/fields.py

```diff
@@ -731,7 +747,31 @@
     },
     "cofire_fuels": {
         "type": "boolean",
-        "description": "Can the generator co-fire fuels?.",
+        "description": "Can the generator co-fire fuels?",
+    },
+    "cofire_energy_source_1": {
+        "type": "string",
+        "description": "Type of fuel.",
+    },
+    "cofire_energy_source_2": {
+        "type": "string",
+        "description": "Type of fuel.",
+    },
+    "cofire_energy_source_3": {
+        "type": "string",
+        "description": "Type of fuel.",
+    },
+    "cofire_energy_source_4": {
+        "type": "string",
+        "description": "Type of fuel.",
+    },
+    "cofire_energy_source_5": {
+        "type": "string",
+        "description": "Type of fuel.",
+    },
+    "cofire_energy_source_6": {
+        "type": "string",
+        "description": "Type of fuel.",
```

Comment:
Should probably specify this description a little further to something like: "The codes representing the type of fuel that will be co-fired"